src/main/kotlin/calculator/model/Numbers.kt

```diff
@@ -0,0 +1,17 @@
+package calculator.model
+
+class Numbers private constructor(
+    private val numbers: List<Number>
+) {
+
+    fun sum(): Int {
+        return numbers.sumOf { it.number }
+    }
```

Comment:
이 부분에 코틀린 감수성을 조금 추가해보는건 어떨까요!
Numbers의 sum() 메서드는 결국 Numbers의 총 합이라는 상태를 나타내는것으로 보입니다!
코틀린에서 val는 상태를 나타내는 프로퍼티인데요 지금 Numbers는 불변객체이기도 하니 아래와 같이 표현하는 방식이 있을 것 같습니다!

불변객체라면
```suggestion
    val sum: Int = numbers.sumOf { it.number }
```

가변객체라면
```suggestion
    val sum: Int
        get() = numbers.sumOf { it.number }
```